client/state/i18n/util.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * External dependencies
+ */
+import { kebabCase, upperCase } from 'lodash';
+
+/**
+ * Generates (naively) a locale in the format ll-CC.
+ * Result may or may not be valid.
+ *
+ * @param  {String} lang WP.com lang/locale code
+ * @param  {String} countryCode GeoIP country code
+ * @return {String}             Naive concat of lang + countryCode
+ */
+export const generateLangCcLocale = ( lang, countryCode ) => {
+	return kebabCase( lang ) + '-' + upperCase( countryCode );
```

Comment:
I'm a bit confused by this function. If a pt-br speaker visits New York or uses a proxy, aren't we going to generate pt-US here, and then miss pt-br in `requestLocalizedPaymentMethods()` above?

Also, Its simple and specific enough that it might fit better inside `requestLocalizedPaymentMethods()`. We will probably end up doing some fancy logic to figure around working out which payment methods are appropriate to offer, but I'd emphasize simplicity for now.